services/user/CommonApi/common/packages/common-lib/src/messaging/FunctionCallRequest.ts

```diff
@@ -12,6 +12,15 @@ export interface QualifiedFunctionCallArgs extends FunctionCallArgs {
     plugin: string;
 }
 
+export const isQualifiedFunctionCallArgs = (obj: any): obj is QualifiedFunctionCallArgs => {
+    return typeof obj === 'object' && obj !== null
+        && typeof obj.service === 'string'
+        && typeof obj.plugin === 'string'
+        && (typeof obj.intf === 'string' || obj.intf === undefined)
+        && typeof obj.method === 'string'
+        && Array.isArray(obj.params);
+}
+
```

Comment:
One day we should look into zod in this section, so we could instead do something like this...

```
const QualifiedFunctionCallArgsSchema = z.object({
    service: z.string(),
    plugin: z.string().optional(),
    intf: z.string(),
    method: z.string(),
    params: z.array()
})

export type QualifiedFunctionCallArgs = z.infer<typeof QualifiedFunctionCallArgsSchema>

export const isQualifiedFunctionCallArgs = (obj: any): obj is QualifiedFunctionCallArgs => {
    return QualifiedFunctionCallArgsSchema.safeParse(obj).success
}
```